docs/tutorials/multi-d-datasets.ipynb

```diff
@@ -296,7 +296,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(d)\n",
+    "d\n",
```

Comment:
I think this will not show the `__repr_html__` if `d` is not the last line in the cell. You will need to use `to_html` if you want to plot after. Alternatively, plot first.

Reply:
Ah, is this because `plot` has a return value?